torchvision/transforms/v2/_geometry.py

```diff
@@ -26,16 +26,16 @@
 
 
 class RandomHorizontalFlip(_RandomApplyTransform):
-    """[BETA] Horizontally flip the given image/box/mask randomly with a given probability.
+    """[BETA] Horizontally randomly flip the given image/video/boxes/mask with a given probability.
 
     .. betastatus:: RandomHorizontalFlip transform
 
-    If the image is torch Tensor, it is expected
-    to have [..., H, W] shape, where ... means an arbitrary number of leading
-    dimensions
+    If the input is a ``torch.Tensor`` or a ``Datapoint`` (e.g. ``Image``, ``Video``, ``BoundingBox`` etc)
+    it can have arbitrary number of leading dimensions. For example,
+    the image can have ``[..., C, H, W]`` shape. A bounding box can have ``[..., N, 4]`` shape.
```

Comment:
Are you sure about the requirement for `N` of bounding boxes? Our tests say 

https://github.com/pytorch/vision/blob/ed48bb1c8c5cbb4dfaf18f4c27a47c42ce2bde27/test/test_transforms_v2_functional.py#L177-L179

so basically only requiring `[..., 4]`.